slice.go

```diff
@@ -592,3 +593,16 @@ func IsSortedByKey[T any, K constraints.Ordered](collection []T, iteratee func(i
 
 	return true
 }
+
+// ToSorted return a sorted copy of a slice
+// Play: https://go.dev/play/p/G4-aR2Yxh3M
+func ToSorted[T constraints.Ordered](collection []T) []T {
+	if collection == nil {
+		return nil
+	}
+
+	copied := make([]T, len(collection))
+	copy(copied, collection)
+	slices.Sort(copied)
+	return copied
+}
```

Comment:
I would have used "Sort"

```suggestion
// Sort return a sorted copy of a slice
// Play: https://go.dev/play/p/G4-aR2Yxh3M
func Sort[T constraints.Ordered](collection []T) []T {
	if collection == nil {
		return nil
	}

	copied := make([]T, len(collection))
	copy(copied, collection)
	slices.Sort(copied)
	return copied
}
```